### PR TITLE
Add function for redelivering webhooks

### PR DESCRIPTION
**Pull request by DeviousLab**
Aims to resolve issue #2060. Let me know if I've done it wrong!

Signed-off-by: DeviousLab <deviouslab@gmail.com>

Fixes: #2060.

### CODE REVIEW

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @DeviousLab !
This is a good start. Please add a new unit test for the new method and make sure to run `gofmt` on your added code.

File: github/orgs_hooks_deliveries.go
Comment (gmlewis) [inline github/orgs_hooks_deliveries.go:61]:
```suggestion
	req, err := s.client.NewRequest("POST", u, nil)
```

**Review by gmlewis — APPROVED** [review]
Thank you, @DeviousLab !
LGTM.

Awaiting second LGTM before merging.

**Review by Parker77 — APPROVED** [review]
LGTM.

**Comment by gmlewis** [comment]
Thank you, @Parker77 !
Merging.

**Pull request merged**
